packages/frontend/src/components/UserSettings/AccessTokensPanel/CreateTokenModal.tsx

```diff
@@ -0,0 +1,202 @@
+            ? new Date(
+                  currentDate.setDate(
+                      currentDate.getDate() + Number(expiresAt),
+                  ),
+              )
+            : undefined;
+        setExpireDate(dateWhenExpires);
+
+        createAccessToken({
+            description,
+            expiresAt: dateWhenExpires,
+            autoGenerated: false,
+        });
+    });
+
+    return (
+        <Modal
+            size="lg"
+            centered
+            opened
+            onClose={() => {
+                onBackClick();
+            }}
+            title={
+                <Title order={4}>
+                    {data ? 'Your token has been generated' : 'New token'}
+                </Title>
+            }
+        >
+            {!isSuccess ? (
+                <form onSubmit={handleOnSubmit}>
+                    <Stack spacing="md">
+                        <TextInput
+                            label="What’s this token for?"
+                            disabled={isLoading}
+                            placeholder="Description"
+                            required
+                            {...form.getInputProps('description')}
+                        />
+
+                        <Select
+                            withinPortal
+                            defaultValue={expireOptions[0].value}
+                            label="Expiration"
+                            data={expireOptions}
+                            required
+                            disabled={isLoading}
+                            {...form.getInputProps('expiresAt')}
+                        ></Select>
+
+                        <Button type="submit" ml="auto" loading={isLoading}>
+                            Generate token
+                        </Button>
+                    </Stack>
+                </form>
+            ) : (
+                <Stack spacing="md">
+                    <TextInput
+                        id="invite-link-input"
+                        label="Token"
+                        readOnly
+                        className="cohere-block sentry-block fs-block"
+                        value={data.token}
+                        rightSection={
+                            <CopyButton value={data.token}>
+                                {({ copied, copy }) => (
+                                    <Tooltip
+                                        label={copied ? 'Copied' : 'Copy'}
+                                        withArrow
+                                        position="right"
+                                    >
+                                        <ActionIcon
+                                            color={copied ? 'teal' : 'gray'}
+                                            onClick={copy}
+                                        >
+                                            {copied ? (
+                                                <IconCheck size="1rem" />
```

Comment:
use <MantineIcon icon={IconCheck} /> and you can use mantine sizing there like size='md'

Reply:
![image](https://github.com/lightdash/lightdash/assets/7611706/0592880f-d440-4172-8d0f-478a68fc4e64)